cmd/start.go

```diff
@@ -70,11 +74,32 @@ $ %s start demo-path2 --max-tx-size 10`, appName, appName)),
 				return err
 			}
 
+			eg := new(errgroup.Group)
+			eg.Go(func() error {
+				for {
+					var sleepTime time.Duration
+					if err := retry.Do(func() error {
+						sleepTime, err = UpdateClientsFromChains(c[src], c[dst])
+						if err != nil {
+							return err
+						}
+						return nil
+					}, retry.Attempts(5), retry.Delay(time.Millisecond*500), retry.LastErrorOnly(true)); err != nil {
+						return err
+					}
+					time.Sleep(sleepTime)
```

Comment:
```suggestion
					time.Sleep(sleepTime-thresholdTime)
```

We should only have notion of threshold time up here in my opinion.